project/Dependencies.scala

```diff
@@ -111,12 +112,8 @@ object Dependencies {
    * Commercial components
    */
 
-  val akkaResilienceEnhancements = Seq(
-    "com.lightbend.akka" %% "akka-diagnostics" % AkkaEnhancements
-  )
-
   // sbt-cinnamon plugin does not have artifacts for Scala 2.13.x
   val telemetry = com.lightbend.cinnamon.sbt.Cinnamon.library.modules.filterNot(_.name.equals("sbt-cinnamon"))
 
-  val allCommercialLibs = akkaResilienceEnhancements ++ telemetry
+  val allCommercialLibs = telemetry
```

Comment:
We should also update the Cinnamon version when that is released.
Therefore I will publish this first as a `22.10.0-M1`